### PR TITLE
[FEATURE] Document Content Block Examples

### PR DESCRIPTION
**Pull request by linawolf**
Releases: main, 13.4

### CODE REVIEW

File: Documentation/ContentBlocks/Index.rst [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+..  _content-blocks-jumbotron-template:
+
+The jumbotron template
+----------------------
+
+The frontend template for the Content Block "Jumbotron" is a normal Fluid
+template. You already used Fluid for the
+`Page templates <https://docs.typo3.org/permalink/t3sitepackage:fluid-templates>`_
+and to adjust the templates of Fluid-Styles Content elements in chapter
```
Comment (nhovratov) [inline Documentation/ContentBlocks/Index.rst:126]: Fluid Styled

File: Documentation/ContentBlocks/Index.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+one of type "Jumbotron".
+
+You can now create a jumbotron with a button and a link.
+
+Directory :path:`packages/my_site_package/ContentBlocks/ContentElements` contains
+one directory for each Content Block that can be used as normal Content Elements.
+
+..  _content-blocks-jumbotron-directory:
+
+Directory structure of a content block
```
Comment (nhovratov) [inline Documentation/ContentBlocks/Index.rst:65]: I'd like to write Content Block consistently in upper camel case, as it is a special name.

File: Documentation/ContentBlocks/Index.rst [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    The new Content Blocks "Jumbotron" and "Carousel"
+
+You can now create a jumbotron with a button and a link.
+
+Directory :path:`packages/my_site_package/ContentBlocks/ContentElements` contains
+one directory for each Content Block that can be used as normal Content Elements.
+
+..  _content-blocks-jumbotron-directory:
+
+Directory structure of a content block
```
Comment (linawolf) [inline Documentation/ContentBlocks/Index.rst:70]:
```suggestion
Directory structure of a Content Block
```

File: Documentation/ContentBlocks/Index.rst [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+..  _content-blocks-jumbotron-template:
+
+The jumbotron template
+----------------------
+
+The frontend template for the Content Block "Jumbotron" is a normal Fluid
+template. You already used Fluid for the
+`Page templates <https://docs.typo3.org/permalink/t3sitepackage:fluid-templates>`_
+and to adjust the templates of Fluid-Styles Content elements in chapter
```
Comment (linawolf) [inline Documentation/ContentBlocks/Index.rst:131]:
```suggestion
and to adjust the templates of Fluid-Styled Content elements in chapter
```

File: Documentation/ContentBlocks/Index.rst [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+------------------------------------------------
+
+File :path:`packages/my_site_package/ContentBlocks/ContentElements/jumbotron/config.yaml`
+defines what fields should be available for the Content Block in the backend:
+
+..  literalinclude:: /CodeSnippets/my_site_package/ContentBlocks/ContentElements/jumbotron/config.yaml
+    :caption: packages/my_site_package/ContentBlocks/ContentElements/jumbotron/config.yaml
+    :linenos:
+
+Each content block must have a unique name with a prefix of your choice that
```
Comment (nhovratov) [inline Documentation/ContentBlocks/Index.rst:110]:
```suggestion
Each Content Block must have a unique name with a prefix of your choice that
```

File: Documentation/ContentBlocks/Index.rst [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+Set up the extension and delete all caches:
+
+..  code-block:: bash
+
+    ddev typo3 extension:setup
+    ddev typo3 cache:flush
+
+..  _content-blocks-jumbotron:
+
+The jumbotron content block
```
Comment (nhovratov) [inline Documentation/ContentBlocks/Index.rst:48]:
```suggestion
The jumbotron Content Block
```

File: Documentation/ContentBlocks/Index.rst [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+
+Directory :path:`packages/my_site_package/ContentBlocks/ContentElements` contains
+one directory for each Content Block that can be used as normal Content Elements.
+
+..  _content-blocks-jumbotron-directory:
+
+Directory structure of a Content Block
+--------------------------------------
+
+A content block consists of a configuration (:file:`config.yaml`), a template and
```
Comment (nhovratov) [inline Documentation/ContentBlocks/Index.rst:73]:
```suggestion
A Content Block consists of a configuration (:file:`config.yaml`), a template and
```

File: Documentation/ContentBlocks/Index.rst [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+
+..  _content-blocks-next:
+
+Next steps
+==========
+
+*   The generated site package contains a second, more elaborate example in
+    directory :path:`packages/my_site_package/ContentBlocks/ContentElements/carousel`.
+*   Learn how to use the `Kickstart command <https://docs.typo3.org/permalink/friendsoftypo3/content-blocks:cb-skeleton>`_
+    to create your own content blocks.
```
Comment (nhovratov) [inline Documentation/ContentBlocks/Index.rst:158]:
```suggestion
    to create your own Content Blocks.
```

**Review by nhovratov — APPROVED** [review]

**Pull request merged**